README.md

```diff
@@ -1,289 +1,590 @@
-* <kbd>EXPERIMENTAL</kbd>**bindingRole** - the name of the role needed to read the binding data exposed from this bindable service.  Implementations of this specification **SHOULD** enforce that only those with the appropriate roles are allowed to access the binding data.
-
-Extra binding properties **SHOULD** also be defined by the bindable service, using one of the patterns defined in [Pointer to binding data](#pointer-to-binding-data).
-
-The data that is injected or mounted into the container may have a different name because of a few reasons:
-* the backing service may have chosen a different name (discouraged, but allowed).
-* a custom name may have been chosen using the `dataMappings` portion of the `ServiceBinding` CR.
-* a prefix may have been added to certain items in `ServiceBinding`, either globally or per service.  
-
-### Request service binding
-
-Binding is requested by the consuming application, or an entity on its behalf such as the [Runtime Component Operator](https://github.com/application-stacks/runtime-component-operator), via a custom resource that is applied in the same cluster where an implementation of this specification resides.
-
-Since the reference implementation for most of this specification is the [Service Binding Operator](https://github.com/redhat-developer/service-binding-operator) we will be using the `ServiceBinding` CRD, which resides in [this folder](https://github.com/redhat-developer/service-binding-operator/tree/master/deploy/crds), as the entity that holds the binding request.  
-
-**Note** - a few updates / enhancements are being proposed to the current `ServiceBinding` CR, tracked [here](https://github.com/application-stacks/service-binding-specification/issues/16#issuecomment-605309629).
-
-Sample CR
+## Resource Type Schema
+
+```yaml
+status:
+  binding:
+    name:  # string
+```
+
+## Example Resource
+
+```yaml
+...
+status:
+  ...
+  binding:
+    name: production-db-secret
 ```
+
+## Well-known Secret Entries
+
+Other than the required `kind` entry and the recommended `provider` entry, there are no other reserved `Secret` entries.  In the interests of consistency, if a `Secret` includes any of the following entry names, the entry value **MUST** meet the specified requirements:
+
+| Name | Requirements
+| ---- | ------------
+| `host` | A DNS-resolvable host name or IP address
+| `port` | A valid port number
+| `uri` | A valid URI as defined by [RFC3986](https://tools.ietf.org/html/rfc3986)
+| `username` | A string-based username credential
+| `password` | A string-based password credential
+| `certificates` | A collection of PEM-encoded X.509 certificates, representing a certificate chain used in mTLS client authentication
+| `privateKey` | A PEM-encoded private key used in mTLS client authentication
+
+`Secret` entries that do not meet these requirements **MUST** use different entry names.
+
```

Comment:
I don't know how to suggest multiline code blocks, so this will need to be manually edited

```suggestion

## Example Secret

apiVersion: v1
kind: Secret
metadata:
  name: production-db
stringData:
  kind: mysql
  provider: bitnami
  host: localhost
  port: 3306
  username: root
  password: root

```

Reply:
Added manually.